src/api/curriculumApi.ts

```diff
@@ -49,6 +49,12 @@ interface Reference {
   tittel: string;
 }
 
+interface CompetenceAimSet extends CompetenceAim {
```

Comment:
Ingen praktisk betydning, men det hadde vore meir logisk å lage et interface LK06Object eller nokke sånt og la både CompetanceAim og CompetanceAimSet arve det. Bruker du forresten psis og upstreamId til nokke? Eller competenceAims? Ser ikkje kor.

Reply:
Bruker ikke `psis` eller `upstream` til noe nei, men tenkte at når jeg først lager en type for objektet, så typer jeg det fullt ut.

Reply:
Ser ikkje heilt behovet for å type opp verdier her som ikkje trengs.